pkg/controllers/job/plugins/ssh/ssh_test.go

```diff
@@ -29,15 +29,27 @@ func TestSSHPlugin(t *testing.T) {
 		params         []string
 		noRoot         bool
 		sshKeyFilePath string
+		sshPrivateKey  string
+		sshPublicKey   string
 	}{
 		{
 			name:           "no params specified",
 			sshKeyFilePath: SSHAbsolutePath,
+			sshPrivateKey: "",
+			sshPublicKey: "",
 		},
 		{
 			name:           "--ssh-key-file-path=/a/b",
 			params:         []string{"--ssh-key-file-path=/a/b"},
 			sshKeyFilePath: "/a/b",
+			sshPrivateKey: "",
```

Comment:
Please execute `go fmt [files]` to make CI happy

Reply:
fixed, thx.